packages/amplify-category-hosting/__tests__/lib/S3AndCloudFront/helpers/cloudfront-manager.test.js

```diff
@@ -0,0 +1,75 @@
+const cloudFrontManager = require('../../../../lib/S3AndCloudFront/helpers/cloudfront-manager'); 
+
+describe('cloudfront-manager', () => {
+    const mockAmplifyMeta = {
```

Comment:
do you need to be of this shape for test?

Reply:
I'm using a real meta data